### PR TITLE
Awaiting Clarification Journey

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-5731 - Landing Page: Petitioner more information needed](https://tools.hmcts.net/jira/browse/DIV-5731)
 - Add new progress bar page section for Awaiting Clarification
 - Created common component to show courts feedback
 - Added feature toggle `awaitingClarification`

[DIV-5732 - Update 'respond to court's feedback' page after user testing](https://tools.hmcts.net/jira/browse/DIV-5732)
 - Add new page court feedback to capture petitioner response
 - Include common component to show courts feedback

[DIV-5733 - Do you want to submit additional docs? page](https://tools.hmcts.net/jira/browse/DIV-5733)
 - Updated content for question
 - Include common component to show courts feedback

[DIV-5736 - Clarification upload documents page](https://tools.hmcts.net/jira/browse/DIV-5736)
 - Updated content
 - Updated upload icon 

[DIV-5232 - More information - check your answers page](https://tools.hmcts.net/jira/browse/DIV-5232)
 - Added specific section for `statementOfTruth`
 - Include common component to show courts feedback

[DIV-5233 - Response Submitted (after clarification) done page](https://tools.hmcts.net/jira/browse/DIV-5233)
 - Add files to done page
 - Add specific content for awaiting clarification journey
 - Updated style for sidebar to be inline with prototype

**NOTE:** This could have been split into separate PR's, however for the following reasons i decided to keep as one PR:
- Work had already been completed on previous PR's so to pick up i just took the latest PR which based the previous PR's
- If i did split it out in order for the build to pass on individual PR's i would have to update unit tests which get updated in proceeding PR's ( waist of time )
- In order to test the proceeding PR's they would each be based of the previous PR and therefor be a pain to merge

### CODE REVIEW

**Comment by dtaylor7** [comment]
Unit tests to follow

File: views/components/clarification-feedback.njk [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{#
+  calfication feedback
```
Comment (yarpo) [inline views/components/clarification-feedback.njk]: clarification feedback?

Comment (dtaylor7) [reply views/components/clarification-feedback.njk]: 👍

File: config/custom-environment-variables.yml [hunk 2]
```diff
@@ -88,6 +88,7 @@ journey:
 features:
   idam: FEATURE_IDAM
   webchat: FEATURE_WEBCHAT
+  awaitingClarificationNew: FEATURE_AWAITING_CLARIFICATION
```
Comment (qzhou-hmcts) [inline config/custom-environment-variables.yml]: can we call this feature flag 'FEATURE_DN_REFUSAL'? I thing we are releasing rejection and clarification at the same time and is the flag we are using in the BE

Comment (dtaylor7) [reply config/custom-environment-variables.yml]: the state is awaiting clarification? maybe they mean different things on BE to FE? but this journey should be active when state is awaiting clarification. i think it makes sense on FE at least anyway

Comment (qzhou-hmcts) [reply config/custom-environment-variables.yml]: yes the state is awaiting clarification, but the whole release functionality is REFUSAL, In other case you'll need another  flag for rejection flow. And in my understanding  clarification and rejection will be release together unless you have different option in mind

**Pull request merged**